lib/rules/mustache-interpolation-spacing.js

```diff
@@ -43,6 +43,22 @@ module.exports = {
         const spaces = rightToken.loc.start.column - leftToken.loc.end.column
         const noSpacesFound = spaces === 0
 
+        if (optSpaces && spaces > optSpaces) {
```

Comment:
Wouldn't simple:
```
if(optSpaces === noSpacesFound || (optSpaces && spaces > optSpaces)) {
```
be enough?

Reply:
yep - just tested your suggestion. pushing changes now!